docs/en/modules/crypto.md

```diff
@@ -191,6 +214,28 @@ The masked message, as a binary string. Use [`crypto.toHex()`](#cryptotohex) to
 print(crypto.toHex(crypto.mask("some message to obscure","X0Y7")))
 ```
 
+## crypto.rnd32()
+
+Provides access to the hardware random number generator on the ESP.
+
+#### Syntax
+`random = crypto.rnd32()`
+
+#### Parameters
+none
+
+#### Return
+A 32bit (unsigned) random value, sourced from the hardware random number generator.
```

Comment:
Is this really unsigned? In the integer build, lua only supports signed 32-bit integers.